LICENSE_DANSE.txt

```diff
@@ -0,0 +1,41 @@
+This program is part of the DiffPy and DANSE open-source projects
```

Comment:
This can stay as it is.  Probably not worth changing to rst.

Reply:
done